drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -19,6 +38,45 @@ def ssn_serial
 
   SORT_OPTIONS = [:last_name_asc, :last_name_desc].freeze
 
+  def self.client_search(input:, user: nil)
+    # Apply ID searches directly, as they can only ever return a single client
+    return searchable_to(user).where(id: input.id) if input.id.present?
+    return searchable_to(user).where(PersonalID: input.personal_id) if input.personal_id
+    return source_for(destination_id: input.warehouse_id, user: user) if input.warehouse_id
+
+    # Build search scope
+    scope = GrdaWarehouse::Hud::Client.where(id: searchable_to(user).select(:id))
+    if input.text_search.present?
+      scope = text_searcher(input.text_search, client_scope: scope) do |where|
+        where(where).pluck(:id)
+      end
```

Comment:
I tested this out, and the scope isn't actually being applied. I think it should be
```suggestion
      scope = text_searcher(input.text_search, client_scope: scope) do |where|
        scope.where(where).pluck(:id)
      end
```

Reply:
<img width="1299" alt="Screen Shot 2022-08-01 at 5 40 38 PM" src="https://user-images.githubusercontent.com/5333982/182251124-6e7e7648-0783-43db-b9a2-082f2907dcf0.png">

Reply:
Thanks for catching that!

Reply:
Fixed with https://github.com/greenriver/hmis-warehouse/pull/2341/commits/98097f76f072be7b32bc29f418a9550fba6aa662